.gitlab-ci.yml

```diff
@@ -45,8 +45,12 @@ build:windows:
     - rmdir /s /q build
     - md build
     - cd build
-    #- cmake ../Plugin
-    #- make
+    - set builddir %cd%
+    - for /f "usebackq tokens=*" %i in (`"%ProgramFiles(x86)%\Microsoft Visual Studio\Installer\vswhere.exe" /latest /property installationPath`) do (set VSDIR=%i)
+    - call "%VSDIR%\Common7\Tools\VsDevCmd.bat"
+    - cd /d %builddir%
+    - msbuild ..\Plugin\abci.sln /t:Build /p:Configuration=Master /p:Platform=x64 /m /nologo /p:OutDir=%builddir%\_out\ /p:IntDir=%builddir%\_tmp\
```

Comment:
Can you add directions on how to update the abci.sln file to the readme?

Reply:
what kind of information are you looking for?

the paths to includes and libs are in the abci.vcproj